File: Webapi/Models/Computer.cs
using System.Collections.Generic;
using Access.Models;

namespace Webapi.Models
{
    public class Computer : BaseEntity
    {
        public string Name { get; set; }
        public string Ip { get; set; }
        public string OS { get; set; }
        public string Username { get; set; }
        public string DiskSpace { get; set; }
        public string MemoryInfo { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public List<Scheduling> Schedulings { get; set; }
    }
}

File: AngularClient/src/app/models/Computer.ts
import { User } from './User';
import { Scheduling } from './Scheduling';

export interface Computer {
    id: number,
    name: string;
    ip: string;
    os: string;
    username: string;
    diskSpace: string;
    memoryInfo: string;
    userId: string;
    user: User;
    schedulings: Scheduling[];
}
